projects/ng-core-tester/src/app/home/home.component.ts

```diff
@@ -33,41 +34,54 @@ export class HomeComponent {
     private dialogService: DialogService,
     private apiService: ApiService,
     private translateLanguageService: TranslateLanguageService,
-    private alertService: AlertService
-  ) {
-    this.apiData = {
-      relative: this.apiService.getEndpointByType('documents'),
-      absolute: this.apiService.getEndpointByType('documents', true),
-    };
+    private toastrService: ToastrService
+    ) {
```

Comment:
from this line, the code is not well indented, but it's not a problem as it's just the test application.